.gitignore

```diff
@@ -95,5 +95,5 @@ pulsar-client/src/test/java/org/apache/pulsar/client/avro/generated
 
 test-reports/
 
-# Gradle Enterprise
-.mvn/.gradle-enterprise/
```

Comment:
should we include both instead of replacing it?

Reply:
valid point

Reply:
Done